### PR TITLE
Restart queue number everyday

### PR DESCRIPTION
**Pull request by jecihjoy**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/queue/api/dao/impl/QueueEntryDaoImpl.java [hunk 1]
```diff
@@ -80,8 +86,20 @@ public String generateVisitQueueNumber(Location location, Queue queue) {
 		criteriaQueueLocation.add(eq("_ql.uuid", location.getUuid()));
 		criteriaQueueLocation.add(eq("_q.uuid", queue.getUuid()));
 		
+		LocalDate minDate = LocalDate.now();
+		LocalDate maxDate = LocalDate.now().plusDays(1);
+		
+		Date date1 = Date.from(minDate.atStartOfDay(ZoneId.systemDefault()).toInstant());
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/queue/api/dao/impl/QueueEntryDaoImpl.java:92]:
```suggestion
		Date startOfDay = Date.from(minDate.atStartOfDay(ZoneId.systemDefault()).toInstant());
```

File: api/src/main/java/org/openmrs/module/queue/api/dao/impl/QueueEntryDaoImpl.java [hunk 2]
```diff
@@ -80,8 +86,20 @@ public String generateVisitQueueNumber(Location location, Queue queue) {
...
+		Date date2 = Date.from(maxDate.atStartOfDay(ZoneId.systemDefault()).toInstant());
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/queue/api/dao/impl/QueueEntryDaoImpl.java:93]:
```suggestion
		Date endOfDay = Date.from(maxDate.atStartOfDay(ZoneId.systemDefault()).toInstant());
```

Comment (makombe) [reply api/src/main/java/org/openmrs/module/queue/api/dao/impl/QueueEntryDaoImpl.java:93]: A small naming suggestion, otherwise everything LGTM :+1:

**Comment by dkayiwa** [comment]
Did you see the failing build?

**Pull request merged**